src/main/anovos/data_report/basic_report_generation.py

```diff
@@ -62,9 +62,11 @@ def anovos_basic_report(spark, idf, id_col='', label_col='', event_label='', out
     AA_funcs = [correlation_matrix, variable_clustering]
     AT_funcs = [IV_calculation, IG_calculation]
     all_funcs = SG_funcs + QC_rows_funcs + QC_cols_funcs + AA_funcs + AT_funcs
-
-    if local_or_emr == "local":
+    
+    if global_run_type == "local":
         local_path = output_path
+    elif global_run_type == "databricks":
+        local_path = "/dbfs/FileStore/tables/report_stats"
     else:
```

Comment:
```suggestion
    elif global_run_type == "emr":
        ...
    else:
        raise ValueError("Invalid global_run_type")
```

We should only allow supported run types and raise a ValueError to catch typos etc. and explicitly tell the user what's going wrong. Otherwise, they might accidentally type "databircks" and wonder why their workflow doesn't run as expected.

Reply:
Added condition that will raise a ValueError to catch typos